### ISSUE TITLE
W39 Lib Team Call

### ISSUE THREAD

**Issue by nondiremanuel** [bitcoindevkit/.github#78]
:calendar: Tues September 26, 2023 12:00 UTC
:round_pushpin: Discord General Voice Channel

Meeting Notes: 

**Please add comments to this issue to suggest issues and PRs to discuss.**

## General Discussion Items

- [ ] BDK 1.0 milestones and timeline
- [ ] BDK Review Club
- [ ] Any review begs? :pray:
- [ ] Blockers? 🪨

## New Projects and Support

- [ ] bitcoindevkit/bdk#1128: which version/milestone? @notmandatory 

#### Recently merged:
- [x] bitcoindevkit/bdk#1084
- [x] bitcoindevkit/bdk#1070
- [x] bitcoindevkit/bdk#969
#### Changes requested:
- [ ] bitcoindevkit/bdk#1064 : changes requested by LLFourn 
#### Need Review:
- [ ] bitcoindevkit/bdk#1124: examples for #1041
- [ ] bitcoindevkit/bdk#1110: ready to be merged or needs other reviews?

#### In progress:
- [ ] bitcoindevkit/bdk#1041: is it ready to review after the last changes? @evanlinjin 

#### To Do:
- [ ] bitcoindevkit/bdk#1125: new bug; needed for alpha.2 or can we postpone it?

#### Recently merged:
- [x] bitcoindevkit/bdk#1086
#### In progress:
- [x] bitcoindevkit/bdk#1028: needs to be rebased after alpha.2 milestone
- [ ] bitcoindevkit/bdk#1115: still in progress or needs review? @realeinherjar
#### To Do:
- [ ] bitcoindevkit/bdk#1079: Lloyd needs some clarification on the API
- [ ] bitcoindevkit/bdk#1113
- [ ] bitcoindevkit/bdk#1094: to be done after #1113
- [ ] bitcoindevkit/bdk#256
- [ ] bitcoindevkit/bdk#1101
- [ ] bitcoindevkit/bdk#994
- [ ] bitcoindevkit/bdk#1042

### Milestone TBD:
- [ ] bitcoindevkit/bdk#1116
- [ ] bitcoindevkit/bdk#1130
- [ ] bitcoindevkit/bdk#1053
- [ ] bitcoindevkit/bdk#1117: in progress to fix #1112 To be confirmed whether it's in milestone 2
- [ ] bitcoindevkit/bdk#1106
- [ ] bitcoindevkit/bdk#1099
- [ ] bitcoindevkit/bdk#1109: #1064 may need to be merged first

## Upcoming Events

- [ ] [BTC Azores](https://btcazores.com/) Sept 23-24
- [ ] [PlanB Lugano](https://planb.lugano.ch/?lang=it) Oct 20-21

🌅 *[Future Call Notes](https://github.com/bitcoindevkit/.github/issues?q=label%3ACall+is%3Aopen)*

📜 *[Past Call Notes](https://github.com/bitcoindevkit/.github/issues?q=label%3ACall+is%3Aclosed)*

[BDK Milestones]:https://github.com/bitcoindevkit/bdk/milestones
[BDK Core Staging Roadmap]:https://github.com/users/LLFourn/projects/2
[BDK 1.0]:https://github.com/orgs/bitcoindevkit/projects/14
[BDK Maintenance]: https://github.com/orgs/bitcoindevkit/projects/6
[BDK-CLI Roadmap]:https://github.com/orgs/bitcoindevkit/projects/7
[Documentation Roadmap]:https://github.com/orgs/bitcoindevkit/projects/9
[BDK Core Project]:https://github.com/users/LLFourn/projects/2/views/1
